docs/insights-archive-sample/config/podnetworkconnectivitychecks.json

```diff
@@ -0,0 +1,8 @@
+{
+  "TCPConnectError": {
+    "kubernetes-apiserver-endpoint-master-0: failed to establish a TCP connection to 10.74.181.101:6443: dial tcp 10.74.181.101:6443: connect: connection refused": 6,
+    "kubernetes-apiserver-service-cluster: failed to establish a TCP connection to 172.30.101.107:443: dial tcp 172.30.101.107:443: connect: connection refused": 6,
+    "openshift-apiserver-endpoint-master-0: failed to establish a TCP connection to 10.128.0.31:8443: dial tcp 10.128.0.31:8443: connect: connection refused": 3,
+    "openshift-apiserver-service-cluster: failed to establish a TCP connection to 172.30.137.129:443: dial tcp 172.30.137.129:443: i/o timeout": 3
```

Comment:
Wouldn't be more interesting to provide the last time the given problem occurred (than to count the number of occurrence)? Just an idea...

Reply:
At least according to the list of PNCC I got from the QuickLab cluster, it seems (and this is a very wild guess, but that's what the gathered JSON suggests) that these checks are performed in a quick succession and the results of the last ~10 checks are stored. In the file I have, there are 10 successes in each batch of checks and they are always 1 minute apart from each other. But it's true that the failures are much older than just a couple of minutes, so maybe it would be a good idea to add that information.
Perhaps the information from `failures` and `outages` should be separated as well. They seem to be related, but adding them together may cause some duplication because it seems like each message appears 3 times (failure message, outage start message, outage end message).

Reply:
I'm on it! I always forget the sample archive exists when making these changes.